src/test/mocks/jsonrpc/index.ts

```diff
@@ -284,6 +284,7 @@ export const presets = {
           isActive: true,
           name: 'Test Provider',
           description: 'Test Provider',
+          providerId: 1n,
```

Comment:
I don't think we've deployed a version of the registry that has this yet, it's a newer addition